docs/administration/back_office/suggestions.md

```diff
@@ -0,0 +1,32 @@
+---
+description: Customize search suggestions configuration and sources.
+---
+
+# Customize search suggestions
+
+In the Back Office, when typing some text in the search bar, some suggestions about what you could be looking for are made directly under this bar.
+
+## Configuration
```

Comment:
There is also site access aware semantic configuration

Reply:
Done in 21233acc

@kisztof Does https://github.com/ibexa/documentation-developer/pull/2222/files#diff-ec74d8d27848528a1597b553f88307e2bf87a1f66ee16a0cd6a70791b160ad4dR12-R18 wording now suit you?